### DISCUSSION TITLE
Error in documentation for a UPI

### DISCUSSION THREAD

**Discussion by gzcwnk** [okd-project/okd#1579] [General]
https://docs.okd.io/latest/installing/installing_bare_metal/installing-bare-metal.html

"Customize the sample install-config.yaml file template that is provided and save it in the <installation_directory"

there is no sample, where do I find this please?

**Comment by mburke5678** [comment]
@vrutkovs Any thoughts?

**Comment by rvanderp3** [comment]
Is this the one? https://docs.okd.io/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal

**Reply by gzcwnk** [reply]
Hi.  It looks like the one I found via google, but Ive never seen one before.  I assumed it would be in the tar.gz which is normal

**Comment by binnes** [comment]
The sample is further down the page and linked in the right navigation in section **Manually creating the installation configuration file**, subsection [Sample install-config.yaml file for bare metal](https://docs.okd.io/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal).

Maybe we should add a link to the sample where it is referenced in point 2 of the Procedure under [Manually creating the installation configuration file](https://docs.okd.io/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-initializing-manual_installing-bare-metal)?
